test/gtest/ucs/test_config.cc

```diff
@@ -196,6 +242,21 @@ UCS_TEST_F(test_config, parse_default) {
     EXPECT_EQ((unsigned)COLOR_BLACK, opts->coach.rear_seat.color);
     EXPECT_EQ(UCS_CONFIG_ULUNITS_AUTO, opts->vin);
     EXPECT_EQ(200UL, opts->engine.power);
+
+    EXPECT_EQ(1024.0, opts->bw_bytes);
+    EXPECT_EQ(UCS_KBYTE * 1024.0, opts->bw_kbytes);
+    EXPECT_EQ(UCS_MBYTE * 1024.0, opts->bw_mbytes);
+    EXPECT_EQ(UCS_GBYTE * 1024.0, opts->bw_gbytes);
+    EXPECT_EQ(UCS_TBYTE * 1024.0, opts->bw_tbytes);
+
+    EXPECT_EQ(128.0, opts->bw_bits);
+    EXPECT_EQ(UCS_KBYTE * 128.0, opts->bw_kbits);
```

Comment:
pls also add some tests for invalid input

Reply:
will add in separate PR